docs/UserGuide.md

```diff
@@ -80,9 +84,10 @@ Adds a person to the address book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+<box type="tip">
+
+**Tip:** A person can have any number of tags (including 0)
+</box>
```

Comment:
We can decide which box style to use and apply that consistently.
I have a feeling will `seamless` or `light` look more elegant and blend in better with the page but best to seek others' opinion on that as well.